src/wrapper/simulation_object_updating_component.py

```diff
@@ -14,6 +14,26 @@ class SimulationObjectUpdatingComponent(Component):
 
     _simulation_objects = None
 
+    @property
+    def simulation_objects(self) -> List["SimulationObject"]:
+        """Returns a list of all objects in the simulation
```

Comment:
You could import `SimulationObject` from `src.wrapper.simulation_objects` or directly use `src.wrapper.simulation_objects.SimulationObject`. This then makes it possible to omit the "" and enables Pylance to recognize these classes.

Reply:
The problem with that is then you get circular-import failures. That is why I didn't import them